models/user.go

```diff
@@ -1137,3 +1137,19 @@ func UnfollowUser(userID, followID int64) (err error) {
 	}
 	return sess.Commit()
 }
+
+// Get the repos starred by a particular user
+func GetStarredRepos(userID int64) ([]*Repository, error) {
+	sess := x.Where("star.uid=?", userID)
+	if setting.UsePostgreSQL {
```

Comment:
I don't understand. `repository.id` is a  repo id, and `star.uid` is a user id; why are we comparing them? I thought that repo ids and user ids were different things. It also seems that your suggestion ignores the `userID` variable, which doesn't seem right.

Please correct me if I'm wrong.